rollup.config.js

```diff
@@ -1,24 +1,12 @@
 import buble from 'rollup-plugin-buble';
-
-var pkg = require('./package.json');
-var externalDeps = Object.keys(Object.assign({}, pkg.dependencies, pkg.peerDependencies));
-var nodeDeps = ['path'];
-var external = externalDeps.concat(nodeDeps);
+import pkg from './package.json';
 
 export default {
-	input: 'src/index.js',
+	input: './src/index.js',
 	plugins: [ buble({ objectAssign: 'Object.assign' }) ],
-	external: external,
+	external: id => !id.startsWith('.') && !id.startsWith('/'),
```

Comment:
hm, not sure about this change - I think I've liked explicit list of externals better, any thoughts about this?

Reply:
i think windows failing on CI might be somehow connected to this change (not sure about this though)

Reply:
Gonna integrate this feature rollup core.